### PR TITLE
Create Send Data > Druva C2C Source doc

### PR DESCRIPTION
**Pull request by keyur-sumo**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
+---
+id: druva-source
+title: Druva
+sidebar_label: Druva
+description: This document explains how to configure the Druva Cloud-to-Cloud source setup on the Sumo logic environment.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md:5]:
```suggestion
description: Learn how to configure the Druva Cloud-to-Cloud source setup in your Sumo Logic environment.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+Before you begin setting up Druva source, you must meet the following requirements:
+* An administrator account is required for **Druva inSync** Cloud.
+* Enable the **Export Events** option. To know the steps, see the [Configure Druva inSync](#configure-druva-insync-to-export-events) section.
+* Obtain access tokens by [authentication](#authentication).
+
+
+### Configure Druva inSync to export events
+
+Only Druva inSync Cloud administrator can enable the option to export events and define the Events API settings. To get the event logs, follow the steps to enable the **Export Events**:
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md:26]:
```suggestion

:::note
You must be a Druva inSync Cloud administrator to enable the option to export events and define the Events API settings.
:::

To get the event logs, follow the steps to enable the **Export Events**:
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+6. Click in the **Categories to export** box and select the events that you want to export from inSync.
+7. In the **Syslog facility** field, type a value between 1 and 23 to assign a Syslog facility ID for inSync events. The default value is 23.
+8. Click **Save**.
+
+
+### Authentication
+
+Druva supports OAuth 2.0-based authentication for incoming requests. Every use of Druva APIs requires authentication to ensure that only authorized users can interact with Druva APIs.
+
+All the requests to Druva APIs are authenticated using OAuth 2.0 access token which you receive in exchange of every authorization grant request you make. The OAuth token will expire after 30 minutes for [Data Governance Cloud](https://apis.druva.com/) and 15 minutes for [Data Governance GovCloud](https://govcloudapis.druva.com/).
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md:41]:
```suggestion
All requests to Druva APIs are authenticated using OAuth 2.0 access tokens, which you receive in exchange of every authorization grant request you make. The OAuth token will expire after 30 minutes for [Data Governance Cloud](https://apis.druva.com/) and 15 minutes for [Data Governance GovCloud](https://govcloudapis.druva.com/).
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+If the Source has any issues during any one of these states, it is placed in an **Error** state.
+
+When you delete the Source, it is placed in a **Stopping** state. When it has successfully stopped, it is deleted from your Hosted Collector.
+
+On the [Collection page](/docs/manage/health-events#collection-page), the Health and Status for Sources is displayed. Use [Health Events](/docs/manage/health-events.md) to investigate issues with collection. You can click the text in the Health column, such as **Error**, to open the issue in Health Events to investigate.
+
+
+## Setup and configuration Source
+
+When you create a Druva Source, you add it to a Hosted Collector. Before creating the Source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector.md).
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md:72]:
```suggestion
When you create a Druva Source, you add it to a Hosted Collector. Before creating the Source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+| Type | Reason | Retries | Retry Behavior | Health Event Name |
+|:--|:--|:--|:--|:--|
+| ThirdPartyConfig  | Normally due to an invalid configuration. You'll need to review your Source configuration and make an update. | No retries are attempted until the Source is updated. | Not applicable                                                    | ThirdPartyConfigError  |
+| ThirdPartyGeneric | Normally due to an error communicating with the third-party service APIs.                                     | Yes                                                   | The Source will retry indefinitely.                               | ThirdPartyGenericError |
+| FirstPartyGeneric | Normally due to an error communicating with the internal Sumo Logic APIs.                                     | Yes                                                   | The Source will retry indefinitely.                   | FirstPartyGenericError |
+
+
+## JSON configuration
+
+Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [how to use JSON to configure Sources](/docs/send-data/use-json-configure-sources) for details.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md:102]:
```suggestion
Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [How to use JSON to configure Sources](/docs/send-data/use-json-configure-sources) for details. 
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
Please see my style/formatting edits

**Comment by kimsauce** [comment]
@JainM6 @keyur-sumo looks like there's an existing PR for this. Can you please advise? https://github.com/SumoLogic/sumologic-documentation/pull/1424

**Comment by keyur-sumo** [comment]
Please close the old PR.

**Comment by keyur-sumo** [comment]
@JainM6 @kimsauce : I have resolved the review comments. Can you please help us re-review and publish the document again?

**Review by kimsauce — APPROVED** [review]

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+1. **Base URL**. Choose the base url from the drop-down menu as per your requirements — [Data Governance Cloud](https://apis.druva.com) and [Data GovernanceGov Cloud](https://govcloudapis.druva.com).<br/><img src={useBaseUrl('/img/send-data/druva-base-url.png')} alt="druva-base-url.png" width="400"/>
+1. **Client ID**. Enter your Client ID. To get Client ID, follow the instructions from [Create and Manage Druva API Credentials](https://docs.druva.com/Druva_Cloud_Platform/Integration_with_Druva_APIs/Create_and_Manage_API_Credentials).
+1. **Secret Key**. Enter your Secret Key. To get Secret Key, follow the instructions from [Create and Manage Druva API Credentials](https://docs.druva.com/Druva_Cloud_Platform/Integration_with_Druva_APIs/Create_and_Manage_API_Credentials).
+1. When you are finished configuring the Source, click **Save**.
+
+## Error types
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md:93]:
```suggestion
## Error Types
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+
+| Type | Reason | Retries | Retry Behavior | Health Event Name |
+|:--|:--|:--|:--|:--|
+| ThirdPartyConfig  | Normally due to an invalid configuration. You'll need to review your Source configuration and make an update. | No retries are attempted until the Source is updated. | Not applicable                                                    | ThirdPartyConfigError  |
+| ThirdPartyGeneric | Normally due to an error communicating with the third-party service APIs.                                     | Yes                                                   | The Source will retry indefinitely.                               | ThirdPartyGenericError |
+| FirstPartyGeneric | Normally due to an error communicating with the internal Sumo Logic APIs.                                     | Yes                                                   | The Source will retry indefinitely.                   | FirstPartyGenericError |
+
+
+## JSON configuration
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md:104]:
```suggestion
## JSON Configuration
```

**Review by JainM6 — DISMISSED** [review]
Minor formatting changes

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md (hunk 7 above)
Comment (kimsauce) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md:104]: Stet - [headers must be sentence case](https://help.sumologic.com/docs/contributing/style-guide/#capitalization)

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md (hunk 6 above)
Comment (kimsauce) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md:93]: Stet - [headers must be sentence case](https://help.sumologic.com/docs/contributing/style-guide/#capitalization)

**Pull request merged**